examples/notebooks/GalSimTutorialCustomCatalog.ipynb

```diff
@@ -0,0 +1,772 @@
+    "               ('halfLightRadius', 'Semi_Major_Axis*PI()/648000.0', numpy.float),\n",
+    "               ('internalAv', '0.1', numpy.float),\n",
+    "               ('internalRv', '3.1', numpy.float),\n",
+    "               ('galacticRv', '3.1', numpy.float),\n",
+    "               ('sedFilename', 'sedName', str, 100),\n",
+    "               ('magNorm', None)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will create a GalSimCatalog to handle our extended objects"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from lsst.sims.GalSimInterface import GalSimGalaxies\n",
+    "\n",
+    "class SersicGalSimCatalog(GalSimGalaxies):\n",
+    "    sedDir = os.path.join(os.getcwd(),'testSEDs')\n",
+    "    bandpassDir = os.path.join(os.getcwd(),'testBandpasses')\n",
+    "    bandpassNames = ['y']\n",
+    "    bandpassRoot = 'myCustomBandpass_'\n",
+    "    componentList = []\n",
+    "\n",
+    "    #the PSF, nose_and_background, and camera will be copied over from NoisyPointSourceCatalog\n",
+    "    #when we copy the GalSimInterpreter, so we do not need to copy it here\n",
+    "    \n",
+    "    def get_sedFilepath(self):\n",
+    "        # this method would be more complicated if there were sub-directories\n",
+    "        # of sedDir that were not stored in the database\n",
+    "        return self.column_by_name('sedFilename')\n",
+    "    \n",
+    "    def get_galacticAv(self):\n",
+    "        ra = self.column_by_name('raJ2000')\n",
+    "        if len(ra)==0:\n",
+    "            return []\n",
+    "\n",
+    "        ebv = self.column_by_name('EBV')\n",
+    "        return 3.1*ebv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "galDB = SersicCatalogDBObject('testInputCatalogs/cartoonSersic.dat')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "galDB.show_mapped_columns()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below we create images that contain both point sources and extended sources."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "starCat = NoisyPointSourceCatalog(starDB, obs_metadata=obs)\n",
+    "starCat.write_catalog('testOutputCatalogs/just_stars.txt')\n",
+    "\n",
+    "galCat = SersicGalSimCatalog(galDB, obs_metadata=obs)\n",
+    "galCat.copyGalSimInterpreter(starCat)\n",
+    "galCat.write_catalog('testOutputCatalogs/just_galaxies.txt')\n",
+    "\n",
+    "galCat.write_images('testImages/compoundImage')"
+   ]
+  },
+  {
```

Comment:
Do you want to add a line here to maybe clear out the images folder after finishing since these files are so big?